src/map/mobskill.cpp

```diff
@@ -69,9 +69,14 @@ bool CMobSkill::isTwoHour() const
   return m_Flag & SKILLFLAG_TWO_HOUR;
 }
 
+bool CMobSkill::isAttackReplacement() const
+{
+  return m_Flag & SKILLFLAG_REPLACE_ATTACK;
```

Comment:
This is probably from copy/pasting my comments ;)
4 spaces here please

Reply:
Yeah it was, sorry ^^
Thanks for correcting. There's a lot of inconsistency in that file regarding indentation.